### DISCUSSION TITLE
ch6/ch6-01

### DISCUSSION THREAD

**Discussion by giscus** [gopl-zh/gopl-zh.github.com#109] [General]
# ch6/ch6-01

<The Go Programming Language>中文版

https://gopl-zh.github.io/ch6/ch6-01.html

**Comment by runningwa** [comment]
这就是类吧，弄个方法名词，看的我好累

**Reply by KrxkGit** [reply]
其实不太一样，类的一些高级特性这个可没有，比如多态

**Reply by mileswangs** [reply]
java对象里的函数也被称为"method"，一个东西

**Reply by Chlokk** [reply]
感觉挺像是Cpp的函数重载

**Comment by ycs0220** [comment]
打卡

**Comment by ycs0220** [comment]
打卡

**Comment by zyuanlbj** [comment]
方法类似于C#中静态类的扩展方法
